### PR TITLE
MSDKUI-1819: Improve guidance maneuver view

### PR DESCRIPTION
**Pull request by imvjk**
Signed-off-by: Vijay Yadav <5671997+imvjk@users.noreply.github.com>

### CODE REVIEW

File: MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/common/BaseView.java [hunk 1]
```diff
@@ -111,4 +112,24 @@ public void setUnitSystem(UnitSystem unitSystem) {
     public UnitSystem getUnitSystem() {
         return mUnitSystem;
     }
+
+    /**
```
Comment (datasun) [inline MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/common/BaseView.java]: Maybe:

protected void enableSaveInstanceState(boolean isSaveInstanceStateEnabled)?
public boolean isSaveInstanceStateEnabled()?

Then it's more obvious what this is about, or?

File: MSDKUIDemo/MSDKUIApp/src/main/res/layout/guidance_maneuver_fragment.xml [hunk 2]
```diff
@@ -15,8 +15,12 @@
   -->
 
 <com.here.msdkui.guidance.GuidanceManeuverView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="wrap_content"
+    android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:paddingStart="28dp"
```
Comment (tstaszak89) [inline MSDKUIDemo/MSDKUIApp/src/main/res/layout/guidance_maneuver_fragment.xml]: Hard coded value?

File: MSDKUIDemo/MSDKUIApp/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -29,4 +29,5 @@
     <dimen name="current_street_margin_bottom">98dp</dimen>
     <dimen name="current_street_margin_side_portrait">80dp</dimen>
     <dimen name="current_street_margin_side_landscape">88dp</dimen>
+    <dimen name="guidanceManeuverEndMargin">28dp</dimen>
```
Comment (tstaszak89) [inline MSDKUIDemo/MSDKUIApp/src/main/res/values/dimens.xml]: It looks that this value should be used in guidance_maneuver_fragment.xml

File: MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/common/BaseView.java (hunk 1 above)
Comment (imvjk) [reply MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/common/BaseView.java]: well, onSaveInstanceState methods is called from android. and i think we cant enable or disable this call so suggested name is confusing too

Comment (imvjk) [reply MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/common/BaseView.java]: I changed the name according to our discussion

**Pull request merged**